examples/public/index.html

```diff
@@ -24,8 +24,11 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Reveal examples</title>
-  <script defer src="/bundle.js"></script><script defer src="/bundle.js"></script><script defer src="/bundle.js"></script></head>
-  <body>
+    <script defer src="/bundle.js"></script>
+    <script defer src="/bundle.js"></script>
+    <script defer src="/bundle.js"></script>
```

Comment:
Is there any reason this is imported trice? (The formatting was done automatically by my prettier extension)